documentation/modules/configuring/con-considerations-for-data-storage.adoc

```diff
@@ -3,18 +3,18 @@
 // assembly-deployment-configuration-kafka.adoc
 
 [id='considerations-for-data-storage-{context}']
-
 = Data storage considerations
 
-An efficient data storage infrastructure is essential to the optimal performance of Strimzi.
+[role="_abstract"]
+For Strimzi to work well, an efficient data storage infrastructure is essential.
 
 Block storage is required.
 File storage, such as NFS, does not work with Kafka.
 
 Choose from one of the following options for your block storage:
 
 * Cloud-based block storage solutions, such as {aws-ebs}
-* {K8sLocalPersistentVolumes}
+* A mounted {K8sLocalPersistentVolumes}
```

Comment:
I would not add this. the way I would read it is that the user has to go and mount it. That is not really the case. So I think this is a bit confusing.